index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Gerg Capston project</title>
+    <link rel="stylesheet" href="style/style.css">
+</head>
+<body>
+    <div class="page-header padding-x">
+        <ul>
+            <li><a href=""> <img src="imgs/icons/github.png" alt=""></a></li>
+            <li><a href=""> Email</a></li>
+            <li><a href=""> twiter </a></li>
+            <li><a href=""> My Page </a></li>
+            <li><a href=""> Logout </a></li>
+        </ul>
+    </div>
+    <nav class="nav_bar padding-x">
+        <a href="index.html"><img class="my-logo" src="./imgs/icons/logo.png" alt="My Log"></a>        
+        <img class="humburger" src="./imgs/icons/humburger.jpg" alt="Humburger Icon">
+        <div class="close-btn">
+            <p>x</p>
+        </div>
+        <ul>
+            <li><a class="about" href="index.html" >Home</a></li>
+            <li><a class="" href="about.html" >About</a></li>
+            <li><a href="" >Programs</a></li>
+            <li><a href="" >Join</a></li>
+            <li><a href="" >Sponser</a></li>
+            <li><a href="" >News</a></li>
+            <li  class="cc" ><ahref="" >CC campign</a></li>
+        </ul>
+    </nav>
```

Comment:
I think it would be much better if your mobile menu covers the entire screen. This way, users can't see other content when scrolling

<img width="237" alt="image" src="https://user-images.githubusercontent.com/37116322/234415110-ad4cca97-fe9a-4cce-ba4d-c7ac790106cf.png">

Reply:
### 🎁🎁🎁 Thanks a lot, I will try my best to correct it. 🎀🎀🎀